sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/GlobalThroughputControlConfigBuilder.java

```diff
@@ -0,0 +1,79 @@
+public class GlobalThroughputControlConfigBuilder {
+    private final CosmosAsyncContainer controlContainer;
+
+    private Duration controlItemRenewInterval;
+    private Duration controlItemExpireInterval;
+
+    GlobalThroughputControlConfigBuilder(CosmosAsyncClient client, String databaseId, String containerId) {
+        checkNotNull(client, "Client can not be null");
+        checkArgument(StringUtils.isNotEmpty(databaseId), "DatabaseId cannot be null nor empty");
+        checkArgument(StringUtils.isNotEmpty(containerId), "ContainerId cannot be null nor empty");
+
+        controlContainer = client.getDatabase(databaseId).getContainer(containerId);
+    }
+
+    /**
+     * Set the control item renew interval.
+     *
+     * This controls how often the client is going to update the throughput usage of itself
+     * and adjust its own throughput share based on the throughput usage of other clients.
+     *
+     * In short words, it controls how quickly the shared throughput will reload balanced across different clients.
+     *
+     * @param controlItemRenewInterval The control item renewal interval.
+     * @return The {@link GlobalThroughputControlConfigBuilder}
+     */
+    @Beta(value = Beta.SinceVersion.V4_13_0, warningText = Beta.PREVIEW_SUBJECT_TO_CHANGE_WARNING)
+    public GlobalThroughputControlConfigBuilder setControlItemRenewInterval(Duration controlItemRenewInterval) {
+        this.controlItemRenewInterval = controlItemRenewInterval;
+        return this;
+    }
+
+    /**
+     * Set the control item expire interval.
+     *
+     * A client may be offline due to various reasons (being shutdown, network issue... ).
+     * This controls how quickly we will detect the client has been offline and hence allow its throughput share to be taken by other clients.
+     *
+     * @param controlItemExpireInterval The control item expire interval.
+     * @return The {@link GlobalThroughputControlConfigBuilder}
+     */
+    @Beta(value = Beta.SinceVersion.V4_13_0, warningText = Beta.PREVIEW_SUBJECT_TO_CHANGE_WARNING)
+    public GlobalThroughputControlConfigBuilder setControlItemExpireInterval(Duration controlItemExpireInterval) {
+        this.controlItemExpireInterval = controlItemExpireInterval;
+        return this;
+    }
+
+    /**
+     * Validate the throughput global control configuration and create a new throughput global control config item.
+     *
+     * @return A new {@link GlobalThroughputControlConfig}.
+     */
+    @Beta(value = Beta.SinceVersion.V4_13_0, warningText = Beta.PREVIEW_SUBJECT_TO_CHANGE_WARNING)
+    public GlobalThroughputControlConfig build() {
+        if (this.controlItemExpireInterval != null && this.controlItemRenewInterval != null) {
+            if (this.controlItemExpireInterval.getSeconds() < this.controlItemRenewInterval.getSeconds()) {
+                throw new IllegalArgumentException("Expire time should not be smaller than renew interval");
```

Comment:
what if the expireInterval is null and renewInterval is not null?
Arguably, in that case, expireInterval would be less than renewInterval, no ?

Reply:
currently, if renewInterval is not null, then the expire interval will be 2 * renewInterval. But we did not force the other way around. 
But I think we can use the formular @moderakh  suggested to calculate the value if one of them is null: 
2 * controlItemRenewInterval + 1 < controlItemExpireInterval